code/game/mecha/equipment/weapons/laser_weapons.dm

```diff
@@ -40,6 +40,19 @@
 	projectile = /obj/item/projectile/beam/heavylaser
 	fire_sound = 'sound/weapons/lasercannonfire.ogg'
 
+/obj/item/mecha_parts/mecha_equipment/weapon/energy/laser/heavy/auto
+	equip_cooldown = 5
+	name = "\improper Multi-laser Cannon"
+	desc = "Four Solaris Laser cannon welded together to achieve an higher firerate."
+	icon_state = "mecha_auto_laser"
+	energy_drain = 180
+	projectile = /obj/item/projectile/beam/heavylaser
+	projectiles = 8
+	projectiles_per_shot = 2
+	fire_cooldown = -5
```

Comment:
Do not use negitive numbers

Reply:
Fun Fact : `fire_cooldown` is never used! At all! it is `equip_cooldown` that is used as delay between shots.

Reply:
Correct

Reply:
Still put it to 0.